test/e2e/apps/job.go

```diff
@@ -718,10 +718,13 @@ done`}
 
 		ginkgo.By("Creating a job")
 		job := e2ejob.NewTestJob("notTerminate", "exceed-active-deadline", v1.RestartPolicyNever, parallelism, completions, &activeDeadlineSeconds, backoffLimit)
+		// set low to TerminationGracePeriodSeconds to ensure quick pod termination
```

Comment:
```suggestion
		// set TerminationGracePeriodSeconds to a low value to ensure quick pod termination
```

Reply:
Actually, it turned out, in my experiments that this does not help . This is only useful to evict quickly a pod that handles SIGTERM and does not make progress. In this case the terminating pods as mostly in Pending phase. When they enter the Running phase they terminate quickly (seconds)